apps/src/code-studio/utils.js

```diff
@@ -48,3 +48,17 @@ export function updateQueryParam(param, value) {
   }
   window.history.pushState(null, document.title, newLocation);
 }
+
+/**
+ * We indicate that a page is Rtl by setting the dir attribute on the root element
```

Comment:
yeah, I was wondering if that would be the case. I guess it's fine to just keep as is and make sure setRtl updates the DOM as well.

Another thought is to get rid of the `isRtl()` function, so nobody is tempted to use that instead of referring to what's in redux. Then you just inline `isRtl` into wherever `setRtl` gets called. Or in addition, you replace `setRtl()` with `syncRtlFromDOM()` and have that look the attribute up and generate the redux action in one function.